.github/docker-node/10/amazonlinux2/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM amazonlinux:2
+
+ENV NODE_VERSION 10.24.1
+
+# centos needs the user to be root; sudo doesn't work.
+USER root
+# yum returns non-zero exit code (100) if packages available for update
+RUN yum -y check-update || echo "packages available for update"
```

Comment:
Minor, `check-update` isn't needed if the intent is to have `yum install` install the latest available package.